.npmrc

```diff
@@ -1 +0,0 @@
-registry=https://registry.npmjs.org
```

Comment:
what about those who develop locally using `npm` ?

Reply:
you would need yarn for running the tests, but i understand some developers might need this, i'll add it back